asllib/Interpreter.ml

```diff
@@ -24,6 +24,7 @@ open AST
 open ASTUtils
 
 let fatal_from pos = Error.fatal_from pos
+let unsupported_expr e = Error.(fatal_from e (UnsupportedExpr (Dynamic, e)))
```

Comment:
This addition entails a warning "This open statement shadows the value from_pos...".
A quick consists in swapping both lines. 
```suggestion
let unsupported_expr e = Error.(fatal_from e (UnsupportedExpr (Dynamic, e)))
let fatal_from pos = Error.fatal_from pos
```
One can also delete the local definition of fatal_from and use the qualified from `Error.fatal_from` in the rest of the file.